test/application/widgets/theme/tags_field_test.dart

```diff
@@ -0,0 +1,181 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:memo/application/constants/strings.dart' as string;
+import 'package:memo/application/theme/theme_controller.dart';
+import 'package:memo/application/widgets/theme/tags_field.dart';
+
+import '../../../utils/widget_pump.dart';
+
+void main() {
```

Comment:
What about a test that deals with "min" length (or not accepting empty strings)?